html/elements/iframe.json

```diff
@@ -1140,6 +1140,65 @@
               "deprecated": false
             }
           }
+        },
+        "mozbrowser": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/API/Using_the_Browser_API",
```

Comment:
https://developer.mozilla.org/en-US/docs/Mozilla/Gecko/Chrome/API/Browser_API would probably be a better landing page for this?